src/main/java/org/embulk/filter/calcite/PageConverter.java

```diff
@@ -99,4 +100,9 @@ public void jsonColumn(Column column) {
             row[i] = pageReader.getJson(i).toJson();
         }
     }
+
+    @SuppressWarnings("deprecation")
+    private static Instant getInstant(final PageReader pageReader, final int index) {
+        return pageReader.getTimestamp(index).getInstant();
```

Comment:
is this code different with `pageReader.getTimestampInstant(index);`?

Reply:
`getTimestampInstant` is available **only** after v0.10.13 -- it means that it won't work with v0.9.23, which is still a stable version of Embulk. Open-sourced plugins under github.com/embulk should keep working with v0.9.23.